application/libraries/Api/Rest/V1/SchemaFactory/SchemaFactoryQuestion.php

```diff
@@ -3,11 +3,16 @@
 namespace LimeSurvey\Api\Rest\V1\SchemaFactory;
 
 use GoldSpecDigital\ObjectOrientedOAS\Objects\Schema;
+use GoldSpecDigital\ObjectOrientedOAS\Contracts\SchemaContract;
 
 class SchemaFactoryQuestion
 {
-    public function make(): Schema
+    /**
+     * @param ?\GoldSpecDigital\ObjectOrientedOAS\Contracts\SchemaContract[] $properties
+     */
+    public function make(SchemaContract ...$properties): Schema
     {
+        $properties = !empty($properties) ? $properties : [];
```

Comment:
I think using `??` might give shorter code, but it's no biggie.

```php
$props = $props ?? [];
```

Reply:
Yeah you are right I will fix.

Reply:
Fixed.